lib/Service/ParticipantService.php

```diff
@@ -1181,6 +1184,16 @@ public function sendCallNotificationForAttendee(Room $room, Participant $current
 			throw new DoesNotExistException('Room mismatch');
 		}
 
+		$userStatus = $this->userStatusManager->getUserStatuses(array($attendee->getActorId()));
+
+		if (empty($userStatus)) {
+			throw new \InvalidArgumentException('User not found');
+		}
+
+		if ($userStatus[0]->getStatus() === IUserStatus::DND){
```

Comment:
The user having no status is default and okay when they are "offline".
So it should not fail in this case and just continue without checking the user status.
So let's combine both checks to:
```suggestion
		if (isset($userStatus[0]) && $userStatus[0]->getStatus() === IUserStatus::DND) {
```